common/app/common/commercial/hosted/HostedArticlePage.scala

```diff
@@ -10,7 +10,25 @@ case class HostedArticlePage(
   pageName: String,
   pageTitle: String,
   standfirst: String,
-  facebookImageUrl: String
+  standfirstLink: String,
```

Comment:
To make this more generic, shouldn't you rather create a `CustomData` class and then provide it as a `Seq[CustomData]`?